Common/Data/Custom/Estimize/EstimizeConsensus.cs

```diff
@@ -0,0 +1,200 @@
+        /// Type of Consensus (EPS or Revenue)
+        /// </summary>
+        [JsonProperty(PropertyName = "type")]
+        public Type? Type { get; set; }
+
+        /// <summary>
+        /// The mean of the distribution of estimates (the "consensus")
+        /// </summary>
+        [JsonProperty(PropertyName = "mean")]
+        public decimal? Mean { get; set; }
+
+        /// <summary>
+        /// The mean of the distribution of estimates (the "consensus")
+        /// </summary>
+        public override decimal Value => Mean ?? 0m;
+
+        /// <summary>
+        /// The highest estimate in the distribution
+        /// </summary>
+        [JsonProperty(PropertyName = "high")]
+        public decimal? High { get; set; }
+
+        /// <summary>
+        /// The lowest estimate in the distribution
+        /// </summary>
+        [JsonProperty(PropertyName = "low")]
+        public decimal? Low { get; set; }
+
+        /// <summary>
+        /// The standard deviation of the distribution
+        /// </summary>
+        [JsonProperty(PropertyName = "standard_deviation")]
+        public decimal? StandardDeviation { get; set; }
+
+        /// <summary>
+        /// The number of estimates in the distribution
+        /// </summary>
+        [JsonProperty(PropertyName = "count")]
+        public int? Count { get; set; }
+
+        /// <summary>
+        /// The timestamp of this consensus (UTC)
+        /// </summary>
+        [JsonProperty(PropertyName = "updated_at")]
+        public DateTime UpdatedAt { get; set; }
+
+        /// <summary>
+        /// The fiscal year for the release
+        /// </summary>
+        [JsonProperty(PropertyName = "fiscal_year")]
+        public int? FiscalYear { get; set; }
+
+        /// <summary>
+        /// The fiscal quarter for the release 
+        /// </summary>
+        [JsonProperty(PropertyName = "fiscal_quarter")]
+        public int? FiscalQuarter { get; set; }
+
+        /// <summary>
+        /// The timestamp of this consensus (UTC)
+        /// </summary>
+        public override DateTime EndTime => UpdatedAt;
+
+        /// <summary>
+        /// Return the Subscription Data Source gained from the URL
+        /// </summary>
+        /// <param name="config">Configuration object</param>
+        /// <param name="date">Date of this source file</param>
+        /// <param name="isLiveMode">true if we're in live mode, false for backtesting mode</param>
+        /// <returns>Subscription Data Source.</returns>
+        public override SubscriptionDataSource GetSource(SubscriptionDataConfig config, DateTime date, bool isLiveMode)
+        {
+            if (!config.Symbol.Value.EndsWith(".C"))
+            {
+                throw new ArgumentException($"EstimizeConsensus.GetSource(): Invalid symbol {config.Symbol}");
+            }
+
+            var symbol = config.Symbol.Value;
+            symbol = symbol.Substring(0, symbol.Length - 2);
+
+            var source = Path.Combine(
```

Comment:
Maybe we should create a helper method to help enforce structure to these alternative data paths?